Etabs_Adapter/Helpers/Results.cs

```diff
@@ -509,6 +509,57 @@ private static List<string> CheckAndGetCases(cSapModel model, IList cases)
 
 
 
+        #endregion
+
+        #region Other results
+
+        public static List<GlobalReactions> GetGlobalReactions(cSapModel model, IList cases = null)
+        {
+            List<string> loadcaseIds = new List<string>();
+            List<GlobalReactions> globalReactions = new List<GlobalReactions>();
+
+            int resultCount = 0;
+            string[] loadcaseNames = null;
+            string[] stepType = null; double[] stepNum = null;
+            double[] fx = null; double[] fy = null; double[] fz = null;
+            double[] mx = null; double[] my = null; double[] mz = null;
+            double gx = 0; double gy = 0; double gz = 0;
+
+            //Get out loadcases, get all for null list
+            loadcaseIds = CheckAndGetCases(model, cases);
+
+            model.Results.Setup.DeselectAllCasesAndCombosForOutput();
+
+            for (int loadcase = 0; loadcase < loadcaseIds.Count; loadcase++)
+            {
+                if (model.Results.Setup.SetCaseSelectedForOutput(loadcaseIds[loadcase]) != 0)
```

Comment:
Here it looks like you are calling the same thing twice?

would probably change this to something like:

```
                if (model.Results.Setup.SetCaseSelectedForOutput(loadcaseIds[loadcase]) != 0)
                {
                    Engine.Reflection.Compute.RecordWarning("Failed to setup result extraction for case " + loadcaseIds[loadcase]);
                }
```

Reply:
I see, however I think it should be more like in 1565868

Reply:
Ahh, missread the lines, looked the same, sorry about that. That all makes sense now! Will test again and approve and merge if it works!

Reply:
No problem, thanks for testing! :)